### ISSUE TITLE
Asciidoctor-diagram 1.5.17 is incompatible with asciidoctor 1.5.x

### ISSUE THREAD

**Issue by robertpanzer** [asciidoctor/asciidoctor-diagram#224]
When trying to build asciidoctorj-diagram 1.5.17 I ran some integration tests with AsciidoctorJ 1.5.8 containing Asciidoctor 1.5.8.
This test failed with:
```
org.jruby.exceptions.RaiseException: (NoMethodError) undefined method `enable_dsl' for Asciidoctor::Diagram::Extensions::DiagramBlockProcessor:Class
	at org.jruby.RubyBasicObject.method_missing(org/jruby/RubyBasicObject.java:1657)
	at RUBY.included(uri:classloader:/gems/asciidoctor-diagram-1.5.17/lib/asciidoctor-diagram/extensions.rb:90)
	at org.jruby.RubyModule.include(org/jruby/RubyModule.java:2638)
	at RUBY.<class:DiagramBlockProcessor>(uri:classloader:/gems/asciidoctor-diagram-1.5.17/lib/asciidoctor-diagram/extensions.rb:303)
```
But the gemspec indicates that it should still be compatible with asciidoctor 1.5.x.

According to @mojavelinux the code should invoke `use_dsl` instead of `enable_dsl` to maintain compatibility with Asciidoctor 1.5.x.

**Comment by pepijnve** [comment]
Thanks for reporting this quickly @robertpanzer; I'm on it. Adding 1.5.x to the CI builds so this doesn't happen again.

**Comment by pepijnve** [comment]
Compatibility issues fixed in 1.5.18

**Comment by robertpanzer** [comment]
🎉 Thanks
